apps/condo/domains/meter/gql.js

```diff
@@ -64,6 +64,42 @@ const EXPORT_PROPERTY_METER_READINGS_QUERY = gql`
     }
 `
 
+const REGISTER_METERS_MUTATION = gql`
+    mutation registerMeters ($data: RegisterMetersInput!) {
```

Comment:
Omg, it looks so strange. We don't need it in a prod

Reply:
renamed to `registerMetersReadings`